cmd/kube-apiserver/app/encryption/encryption.go

```diff
@@ -14,7 +14,8 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package encryptionconfig
+// Package encryption handles assigning encryption providers to each group resource storage.
+package encryption
```

Comment:
@cheftako and @lavalamp wanted this to be outside`k8s.io/apiserver` because it contains configuration parsing code, which should not reside in a generic library.

Please advise.